### PR TITLE
issue 763 test database setup

### PR DESCRIPTION
**Pull request by baob**
Closes issue #763

- correct link to github readme document
- notes about db migrations and the test db
- add notes on database seeding

### CODE REVIEW

File: app/views/static_pages/about.html.haml [hunk 1]
```diff
@@ -83,6 +83,6 @@
       %li
         Developers are welcome to join the project.  The
 
-        = link_to "README document"
+        = link_to "README document", "https://github.com/swapmyvote/swapmyvote/blob/master/README.md"
```
Comment (aspiers) [inline app/views/static_pages/about.html.haml:86]: Wow good catch, @tdg do we have a Google Search Console or similar set up to catch broken links?

**Review by aspiers — APPROVED** [review]

File: app/views/static_pages/about.html.haml (hunk 1 above)
Comment (tdg) [reply app/views/static_pages/about.html.haml:86]: I don't think we do, no. I will have a look into it tomorrow

Comment (tdg) [reply app/views/static_pages/about.html.haml:86]: haha, precisely a year later, this is now set up

Comment (aspiers) [reply app/views/static_pages/about.html.haml:86]: Yes but also it's possible to set up CI to auto-scan for broken links. I've created #1032 to track this.

**Pull request merged**